### ISSUE TITLE
http://www.dilidili.wang/

### ISSUE THREAD

**Issue by leoxxx** [reek/anti-adblock-killer#3388]
http://www.dilidili.wang/
FIREFOX
GREASEMONKEY
ABP
caused by http://www.dilidili.wang/newjs/main.js

**Comment by uBlock-user** [comment]
this filter works - `dilidili.wang##script:inject(abort-on-property-read.js, $backTop)`

I'm unsure as if that breaks anything.

**Comment by uBlock-user** [comment]
@jspenguin2017

**Comment by jspenguin2017** [comment]
If that script is the problem, then filtering `document.querySelectorAll` is the best way to go. 
The website seems to be down, I'll test later.

**Comment by jspenguin2017** [comment]
Website still down...

**Comment by jspenguin2017** [comment]
Website is up now, filtering `document.querySelectorAll` seems to work for me.

**Comment by leoxxx** [comment]
`dilidili.wang##script:inject(document.querySelector)`
Is this?

**Comment by uBlock-user** [comment]
@leoxxx No, He meant `dilidili.wang##script:inject(abort-on-property-read.js, document.querySelector)`

**Comment by jspenguin2017** [comment]
No, I mean filtering with script rule, it is already in my extension.

**Comment by uBlock-user** [comment]
actually that breaks jquery

**Comment by jspenguin2017** [comment]
It is handled here: https://github.com/jspenguin2017/uBlockProtector/issues/426

**Comment by Fezreal** [comment]
@gorhill  
This is a very worthy question , please find a solution to fix this "bug" 
If the problem can not be resolved, then all the sites will be used in a similar way to stop ublock

**Comment by jspenguin2017** [comment]
It can be solved with "native" nooping filters, but as they can tweak the bait file, to avoid cat-and-mouse game, I went ahead with a more aggressive approach which patches `innerHTML` property.
